tools/apiview/parsers/CONTRIBUTING.md

```diff
@@ -45,66 +57,32 @@ Each tree node has top tokens which should be used to capture the main tokens on
 
-Ensure each node has an Id and Kind. The combination of `Id`, `Kind` and `SubKind` should make the node unique across all nodes in the tree. This is very important. For example a class and a method can potentally have the same Id, but the kind should diffrentiate them from each other.
+### StructuredToken
 
-Dont worry about indentation that will be handeled by the tree structure, unless you want to have indentation between the tokens then use `TabSpace` token kind.
+- Assign the final parsed value to a `List<APITreeNode> APIForest` property of the `CodeFile`
 
+## Serialization
 
-### StructuredToken
-- `Value` : The token value which will be dispalyed. Spacing tokens don't need to have value.
-- `Id` : This is essentially whatever existing parser was asigning to the token DefinitionId. You dont have to assign a tokenId to everytoken.
-- `Kind` *(Required)* : An enum
-  - `Content` Specifies that the token is content
-  - `LineBreak` Space token indicating switch to new line.
-  - `NonBreakingSpace` Regular single space
-  - `TabSpace` 4 NonBreakingSpaces
-  - `ParameterSeparator` Use this between method parameters. Depending on user setting this would result in a singlespace or new line
-  - `Url` A url token should have `LinkText` property i.e `token.Properties["LinkText"]` and the url/link should be the token value.
-  All tokens should be content except for spacing tokens and url. ParameterSeparator should be used between method or function parameters. 
-- `Tags` : Use this for opt in or opt out boolean properties.The currently supported tags are 
-  - `SkippDiff`  Indicate that a token should not be used in computatation of diff.
-  - `Deprecated` Mark a token as deprecated
-- `Properties` : Properties of the token.
-  - `GroupId` : `doc` to group consecutive comment tokens as documentation.
-  - `NavigateToId` Id for navigating to where the node where the token is defined. Should match the definitionId of the referenced node.
-- `RenderClasses` : Add css classes for how the tokens will be rendred. Classes currently being used are `text` `keyword` `punc` `tname` `mname` `literal` `sliteral` `comment` Feel free to add your own custom class. Whatever custom classes you use please provide us the appriopriate css for the class so we can update APIView.
-
-
-Assign the final parsed value to a `List<APITreeNode> APIForest` property of the `CodeFile`
-
-## Serilizations
-
-Serialize the generated code file to JSON them compress the file using Gzip compression. Try to make the json as small as possible by ignoring null values and empty collections.
+Serialize the generated code file to JSON and then compress the file using Gzip compression. Try to make the json as small as possible by ignoring null values and empty collections.
+Don't worry about indentation that will be handled by the tree structure. In the case you want to have indentation between the tokens then use `TabSpace` token kind.
```

Comment:
This statement is not true. Indentation is only "handled for you" when you model things with parent child relationships. Vast amounts of information are expected to be put into "TopTokens" where you will get no such automatic indentation.

Reply:
But isn't that what it says in the later part of the paragraph. Perhaps I can make it clearer.